packages/esm-outpatient-app/src/patient-search/patient-search.component.tsx

```diff
@@ -23,9 +25,13 @@ const PatientSearch: React.FC<PatientSearchProps> = ({ closePanel }) => {
         <div className="omrs-main-content">
           {searchType === SearchTypes.BASIC ? (
             <BasicSearch toggleSearchType={toggleSearchType} />
-          ) : (
+          ) : searchType === SearchTypes.ADVANCED ? (
             <AdvancedSearch toggleSearchType={toggleSearchType} />
-          )}
+          ) : searchType === SearchTypes.SEARCH_RESULTS ? (
+            <SearchResults patients={[]} toggleSearchType={toggleSearchType} />
+          ) : searchType === SearchTypes.SCHEDULED_VISITS ? (
```

Comment:
SCHEDULE_VISIT is not a search type I would extract the logic for displaying scheduled visits away from search type to something like `showScheduleVisit`